### DISCUSSION TITLE
Playtime not updating in client (after first load of profile)

### DISCUSSION THREAD

**Discussion by memesnightmares** [ppy/osu#17229] [Q&A] [answered]
I'm super excited to see lazer playtime updating in web :D However it seems like in the client it only updates on first fetch. So, if I check my playtime in-game, play a map, then then check my playtime in-game again, it won't have changed. However, on the website it will have updated. This sounds like the profile page is being cached a little too aggressively but I have no idea what the actual cause is.

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/13493
